### DISCUSSION TITLE
How to jump to a file from the NeogitCommitView?

### DISCUSSION THREAD

**Discussion by georgeguimaraes** [NeogitOrg/neogit#1232] [Q&A]
Hi,

Let me explain my use case. I move to a branch (usually using git checkout in the terminal), and I want to see the changes/files in the last few commits to remember what the branch is about (or even because I'm terrible to remember the name of files).

I open neogit, I use the Recent Commits fold to select the last commit and when browsing the changes in the NeogitCommitView popup, I'd want to go to the file/line that the hunk is referring to. 

So, in other terms, I want to use Neogit to browse the few last changes made to the branch. Is it possible?

**Comment by AlejandroSanchez90** [comment]
im wondeting the same.. I would like a way to jump to files from my recent commits

**Comment by tmerse** [comment]
This would be super helpfull
